src/routes/OverviewPage/DescriptionsCard.tsx

```diff
@@ -16,21 +16,21 @@ function DescriptionsCard() {
   return (
     <Card>
       <ExpandableDescription
-        title="About Advanced Cluster Security"
+        title="About Red Hat Advanced Cluster Security Cloud Service"
```

Comment:
I think it might be worthwhile to save `Red Hat Advanced Cluster Security Cloud Service` as a constant if the string is being referenced in multiple places.

Reply:
hmm that's a good point! i think for now i'll leave it as is but if another instance appears, i'll factor it out then